check/pytest-and-incremental-coverage

```diff
@@ -19,7 +28,28 @@ cd ${own_directory}
 repo_dir=$(git rev-parse --show-toplevel)
 cd ${repo_dir}
 
-PYTHONPATH="$(pwd)" python dev_tools/run_pytest_and_incremental_coverage.py $@
+# Figure out which branch to compare against.
+rest="$@"
+if [ ! -z "$1" ] && [[ $1 != -* ]]; then
+    branch="$1"
+    rest="${@:2}"
+elif [ ! -z $(git branch --list upstream/master) ]; then
```

Comment:
`git branch --list` only seems to work on local branches.  rev-parse could work.

Reply:
Obsoleted.